goodtablesio/models/job.py

```diff
@@ -110,15 +113,53 @@ def get_ids():
     return [j.id for j in job_ids]
 
 
-def get_all():
-    """Get all jobs in the database as dict.
+def find(filters=None, limit=10, offset=0):
```

Comment:
Encapsulation is great but when it really create new abstraction but:
```
jobs = models.job.find(
         filters=[
             models.job.Job.plugin_name == 'github',
             models.job.Job.plugin_conf[('repository', 'owner')].astext == org,
             models.job.Job.plugin_conf[('repository', 'name')].astext == repo
             ]
     )
```
doesn't seem very different from using vanilla sqlalchemy (wrapper abstraction leak).

Architecturally in this case our dependency graph is `view -> model + wrapper -> db` (view knows about model in any case and it's a vital pattern in any system). Without wrapper `view -> model -> db`.

Just afraid we will run into situation when adding every new job view in this case will lead to updating this function.